src/components/course/enrollment/common.jsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable import/prefer-default-export */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@edx/paragon';
+
+// Common UI components for enrollment to avoid duplication, consistent styling etc.
+// Each enroll component can use the EnrollButtonCta to render the enroll behavior
+
+const EnrollButtonWrapper = ({
+  as: Component,
+  children,
+  ...props
+}) => (
+  <div className="enroll-wrapper" style={{ width: 270 }}>
```

Comment:
Please move the style to a stylesheet and use our available css variables for the width if possible.